fathom/src/driver.rs

```diff
@@ -11,6 +11,8 @@ use crate::source::{ByteRange, FileId, Span, StringInterner};
 use crate::surface::{self, elaboration};
 use crate::BUG_REPORT_URL;
 
+mod files;
```

Comment:
I can see the idea behind putting this under `driver`, but I think it might be better off as a top-level module for now – especially if we decide to move `FileId` into the `files` module (see later review comment).